cmd/server/assets/admin/realms/edit.html

```diff
@@ -24,11 +24,17 @@ <h1>Edit {{$realm.Name}}</h1>
     <div class="card mb-3 shadow-sm">
       <div class="card-header">Details</div>
       <div class="card-body">
-        <div class="alert alert-warning" class="mb-4">
+        <div class="alert alert-warning mb-4">
           Only a subset of realm fields are editable after creation. Work with
           the realm administrator to update these values.
         </div>
 
+        {{if $currentUser.CanAdminRealm $realm.ID}}
```

Comment:
I think I'd prefer to have this in the header as an icon with a tooltip. If the user is not a member of the realm, the action should be disabled with helpful text telling them to join first. Otherwise, the tooltip should say "edit this realm in the realm view" or something.

Reply:
Done